cou/apps/app.py

```diff
@@ -487,19 +485,23 @@ def _get_refresh_charm_plan(
         :rtype: Optional[UpgradeStep]
         """
         switch = None
-        description = (
-            f"Changing '{self.name}' channel from: '{self.channel}' "
-            f"to: '{self.expected_current_channel}'"
-        )
```

Comment:
A default description is still needed at L#514 for charmhub aoplication with channel outside of the possible_current_channels list.

Reply:
Also, could you add a unit test for this case?